manticore/platforms/evm.py

```diff
@@ -1049,6 +1051,41 @@ def __reduce__(self):
         return (self.__class__, (self.data, ))
 
 
+def concretized_args(**policies):
+    '''
+    Make sure an EVM instruction has all of its arguments concretized according to
+    provided policies.
+
+    Example decoration:
+
+        @concretized_args(size='ONE')
+        def LOG(self, address, size, *topics):
+            ...
+
+    The above will make sure that the |size| parameter to LOG is Concretized when symbolic
+    according to the 'ONE' policy.
+
+    :param policies: A kwargs list of argument names and their respective policies.
+                         Provide None or '' as policy to use default.
+    :return: A function decorator
+    '''
+    def concretizer(func):
+        @wraps(func)
+        def wrapper(*args, **kwargs):
+            spec = inspect.getargspec(func)
+            sys.stderr.write("args: {}, kwargs: {}".format(repr(args), repr(kwargs)))
```

Comment:
Or just `'args: {!r}, kwargs: {!r}'.format(args, kwargs)`

btw https://pyformat.info/

Reply:
That was an accidentally-committed debug output, which is now removed